File: Rust/1005.rs
use std::io::{stdin};

fn main() {
    let mut num1 = String::new();
    stdin().read_line(&mut num1).expect("");
    let num1: f64 = num1.trim().parse().expect("");

    let mut num2 = String::new();
    stdin().read_line(&mut num2).expect("");
    let num2: f64 = num2.trim().parse().expect("");

    let media = ((num1 * 3.5) + (num2 * 7.5)) / 11.0;

    println!("MEDIA = {:.5}", media);
}

File: Uri Online Judge/C#/1005.cs
using System;
using System.Globalization;

class URI
{
    static void Main(string[] args)
    {
        double num1 = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
        double num2 = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
        double media = (num1*3.5 + num2*7.5) / 11;
        Console.WriteLine("MEDIA = {0}", media.ToString("0.00000", CultureInfo.InvariantCulture));
    }
}
